e2e/fixtures/schedule.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  allocation: 'The proceedings continue to be allocated to Lewis Birks',
+  application: 'The local authority has applied for a care order',
+  todaysHearing: 'Today\'s case was listed for an INTERIM CARE ORDER HEARING',
```

Comment:
Couldn't we use " and don't escape '? 🤔

Reply:
`  4:18  error  Strings must use singlequote  quotes` 😢

Reply:
Yeah unfortunately linter does not like double quotes :/

Reply:
linter... 👎